roles/sap_hana_preconfigure/README.md

```diff
@@ -1,17 +1,17 @@
 # sap_hana_preconfigure Ansible Role
 
 This role installs additional required packages and performs additional configuration steps for installing and running SAP HANA.
-If you want to configure a RHEL system for the installation and later usage of SAP HANA, you have to first run role sap_general_preconfigure
-and then role sap_hana_preconfigure.
+If you want to configure a RHELsystem for the installation and later usage of SAP HANA, you have to first run role sap_general_preconfigure
+and then role sap_hana_preconfigure.  However, if we wish to run SLES for HANA, you may run only this role.
```

Comment:
@mr-stringer I understand that for SLES, the role sap_general_preconfigure is not needed at all, neither for NetWeaver nor for HANA. Can you think of a good wording for the README.md file in sap_general_preconfigure? Maybe just a single line at the end of the first chapter, just like here?

Reply:
I have added a line to the sap_general_preconfigure readme.

```
For SLES systems, this role may not be necessary.  The majority of SAP preparation and tuning is covered by `saptune` which is configured in the `sap_hana_preconfigure` and `sap_netweaver_preconfigure` roles.
```